internal/distro/rhel85/distro.go

```diff
@@ -212,7 +197,9 @@ type imageType struct {
 	// bootable image
 	bootable bool
 	// If set to a value, it is preferred over the architecture value
-	bootType BootType
+	bootType distro.BootType
+	// List of valid arches for the image type
+	validArches ValidArches
```

Comment:
I have basically the same comment to the comment and structure member name. While it is true that any architectures not listed in the map are not supported by the image type **if** it requires a partition table, the name is misleading. Something more descriptive would be IMO better. E.g. `basePartitionTables` or `defaultPartitionTables`, or anything else that comes to your mind 😉